t/ui/27-plugin_obs_rsync_status_details.t

```diff
@@ -124,24 +124,23 @@ my %params = (
     'Batch1'            => ['191216_150610', 'containers', 'BatchedProj', '470.2, 469.1'],
 );
 
-sub _wait_helper {
-    my ($element, $test_break, $refresh) = @_;
+sub _wait_for_change ($selector, $break_cb, $refresh_cb = undef) {
+
     my $ret;
-    for (0 .. 3) {
-        for (0 .. 30) {
-            $ret = $driver->find_element($element)->get_text();
-            return $ret if $test_break->($ret);    # uncoverable statement
-            sleep .1;                              # uncoverable statement
-        }
+    for my $i (0 .. 30) {
+
         # sometimes gru is not fast enough, so let's refresh the page and see if that helped
-        if ($refresh) {                            # uncoverable statement
-            $refresh->();                          # uncoverable statement
-        }
-        else {
-            $driver->refresh();                    # uncoverable statement
+        if ($i > 0) {
+            sleep 1;
```

Comment:
previously there was `sleep .1`, now it's `sleep 1`. Can this not unnecessarily increase the wait time?

Reply:
That's actually the point. The background jobs are very slow. For Circle CI i actually had to increase the overall timeout from 15 to 30 seconds max.

Reply:
Since this now uses a `wait_for_element` too, i don't think the 1 second increase has a negative impact. The previous more frequent polling is just not needed anymore because of that. Now the `sleep` is only for waiting for background jobs, not the page to be rendered.

Reply:
True, still for an individual test waiting 1 second when it's not needed is wasteful. Is it ok to keep `sleep .1` here even if it's not the same thing we wait for as in before?

Reply:
I just tried it (reducing `sleep` to `.1`) and the result is 100% CPU usage in my VM and the test timing out almost all the time.

Reply:
If the scale we're dealing here with is seconds then `sleep 1` makes sense.

Reply:
Pretty sure the test is faster with `sleep 1`, because with Selenium refreshing the page less often the background jobs get more CPU cycles.

Reply:
Actually, before this patch:
```
ok
All tests successful.
Files=1, Tests=61, 39 wallclock secs ( 0.07 usr  0.01 sys +  3.36 cusr  0.90 csys =  4.34 CPU)
Result: PASS
```
And after:
```
ok
All tests successful.
Files=1, Tests=91, 34 wallclock secs ( 0.09 usr  0.03 sys +  3.17 cusr  0.88 csys =  4.17 CPU)
Result: PASS
```

Reply:
A little counter intuitive, but the longer `sleep` is better for test runtime. 😀

Reply:
Still, a test runtime of more than 30s for just one tiny feature of openQA is rather horrible so this is why I am looking for ways to improve. Granted, that one little statement obviously isn't the problem :D

Reply:
Sorry, i really don't want to work on that plugin to improve performance (and that will be necessary). 😬 More stability will have to be enough for today.

Reply:
Maybe worth mentioning, the whole thing is still just a plugin. So nothing prevents us from moving it into a separate GitHub repo (and RPM package).